tools/replaceColumn/replaceColumn.xml

```diff
@@ -14,35 +14,46 @@ replace_file = '$replace_information'
 original_file = '$original_file'
 column = int("$column_replace") - 1
 ignore_start_lines = int("$skip_lines")
-delimiter_local = "\t" if str("$delimiter") == "" else str("$delimiter")
+delimiter_local = "\t" if len(str("$delimiter")) != 1 else str("$delimiter")
+comment_str = str("$pass_comments")
+unk_strat = str("$unknowns_strategy")
 
-## read conversion information to index 
+## read conversion information to index
 conversion = {}
 
 with open(replace_file, 'r') as conversion_file:
     for line in conversion_file:
         conv_key_value = line.strip().split()
         if len(conv_key_value) == 2:
-            conversion[conv_key_value[0]] = conv_key_value[1]                
+            conversion[conv_key_value[0]] = conv_key_value[1]
 
 ## read file line by line, search for column entry if it can be replaced. Otherwise it will be skipped.
 with open("output_file", 'w') as output:
     with open(original_file) as original:
         for i, line in enumerate(original):
-            if i < ignore_start_lines:
+            if i < ignore_start_lines or (comment_str and line.startswith(comment_str)):
                 output.write(line)
                 continue
 
             if str("$delimiter") == "":
                 line_content = line.split()
+            elif str("$delimiter") == "tab":
+                line_content = line.rstrip().split('\t')
+            elif str("$delimiter") == " ":
+                line_content = line.rstrip().split(' ')
             else:
-                line_content = line.split(str("$delimiter"))
+                line_content = line.rstrip().split(str("$delimiter"))
```

Comment:
I think that this can now be simplified to:
```python
            if str("$delimiter") == "":
                line_content = line.split()
            else:
                line_content = line.rstrip().split(delimiter_local)
```